doc/api/buffer.md

```diff
@@ -542,14 +537,13 @@ Note that the `Buffer` module pre-allocates an internal `Buffer` instance of
 size [`Buffer.poolSize`] that is used as a pool for the fast allocation of new
 `Buffer` instances created using [`Buffer.allocUnsafe()`] (and the deprecated
 `new Buffer(size)` constructor) only when `size` is less than or equal to
-`Buffer.poolSize >> 1` (floor of `Buffer.poolSize` divided by two). The default
-value of `Buffer.poolSize` is `8192` but can be modified.
+`Buffer.poolSize >> 1` (floor of [`Buffer.poolSize`] divided by two).
```

Comment:
Do we really want this "floor of ... divided by two" here? Looks like condition doesn't change if we remove it, `Buffer.poolSize >> 1` is rather computation optimization than something users should be aware of.

For example, if `Buffer.poolSize` would be equal to `3`, both conditions `size is less than or equal to Buffer.poolSize >> 1 (floor of Buffer.poolSize divided by two)` (`size <= 1`) and `size is less than or equal to Buffer.poolSize divided by two` (`size <= 1.5`) are equal for any integer `size`.

Or do we explicitly want to specify behavior for cases when floating number is passed as a `size`?